Client/WebAuthN/U2FExtensions.swift

```diff
@@ -231,8 +231,8 @@ class U2FExtensions: NSObject {
         
         super.init()
         
-        let handleCancelButton: () -> PopupViewDismissType = {
-            self.sendJSError()
+        let handleCancelButton: () -> PopupViewDismissType = { [weak self] in
+            self?.sendJSError()
```

Comment:
this was actually a harder leak, each tab close increased app memory by around 1 megabyte until the app was killed